src/components/filter-selector/FilterSelector.styles.ts

```diff
@@ -11,7 +11,6 @@ export const styles = {
     px: '12px',
```

Comment:
I would like to suggest renaming this component. As it could be used not only as filter.

Reply:
So, in your opinion, which name would be more flexible? AppSelector, for example?

Reply:
You tell me :) it is a button, with menu. Looks like AppButtonMenu. And also we can decompose it to,

AppMenuButton that would have children for menu section

And

AppSelectButton as AppMenuButton with select container for menu

Reply:
I used the useMenu hook to render the menu in this component. I think its good way to make this component more readable and more flexible. And also we use render JSX from hook, not in this component. I made small refactors, that made this component more clean